tests/components/homewizard/snapshots/test_sensor.ambr

```diff
@@ -6717,7 +6717,7 @@
     'previous_unique_id': None,
     'supported_features': 0,
     'translation_key': None,
-    'unique_id': 'homewizard_gas_meter_G001',
+    'unique_id': 'homewizard_DeviceType.GAS_METER_G001',
```

Comment:
Note to self; Have to take a look at this one, this other 'serial_number' and 'unique_id' changes will cause isues.